gravitee-apim-console-webui/src/management/api/documentation-v4/api-documentation-v4.component.spec.ts

```diff
@@ -115,6 +115,33 @@ describe('ApiDocumentationV4', () => {
     jest.resetAllMocks();
   });
 
+  describe('ngOnInit - homepage filter', () => {
+    it('should filter out pages with homepage=true', async () => {
```

Comment:
Test descriptions should be business logic/user experience. Ideally, they even match word-for-word with the ticket's acceptance criteria.

```suggestion
  describe('Homepage section, () => {
    it('should only show current homepage', async () => {
```